prototype1/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "parse": "cd ./../parser && npm start ./.. ./prototype1/src/data.json && cd ../prototype1",
+    "parse": "cd ../parser && npm start .. main ../prototype1/src/data.json && cd ../prototype1",
```

Comment:
It seems the file is written to the outpath relative to the repo path argument, so when I run this, the file is not plased at the proper location. Probably an issue with another change somewhere. I should probably be resolved relative to the current directory instead?